patches/api/0040-LootTable-API.patch

```diff
@@ -238,6 +238,20 @@ index 0000000000000000000000000000000000000000..fd184f13f5e8ee5cf829fff4f44696e1
 +        cancelled = cancel;
 +    }
 +}
+diff --git a/src/main/java/org/bukkit/block/Barrel.java b/src/main/java/org/bukkit/block/Barrel.java
+index aa1bb7a1a1c94b0b029bb60026efbc7f55584dd7..b4430cb7bb8e33016a1af5fcb1cf4f671f364717 100644
+--- a/src/main/java/org/bukkit/block/Barrel.java
++++ b/src/main/java/org/bukkit/block/Barrel.java
+@@ -1,8 +1,6 @@
+ package org.bukkit.block;
+ 
+-import org.bukkit.loot.Lootable;
```

Comment:
Just leave the import, even if it is unused here.

Reply:
I'm assuming it's something to do with reverting patches?

Reply:
Keeping that import adds maintenance burden to those maintaining the project. Leaving it as it was would mean the patch might fail if there's any changes to the imports in that line, even though it has no actual influence in the changes implemented by the patch.

TLDR; the less changes in a diff, the less likely for conflicts to occur, the easier it is to maintain.